packages/dx-vue-grid-bootstrap4/src/templates/table-header-cell/sorting-control.test.js

```diff
@@ -4,12 +4,18 @@ import { SortingControl } from './sorting-control';
 const ENTER_KEY_CODE = 13;
 const SPACE_KEY_CODE = 32;
 
+const defaultProps = () => ({
+  align: '',
+});
+
 describe('SortingControl', () => {
   it('can get focus', () => {
     const tree = shallow({
       render() {
         return (
-          <SortingControl />
+          <SortingControl
+            {...{ attrs: defaultProps() }}
```

Comment:
We are using the following syntax in such cases: `{...{ attrs: { ...defaultProps } }}`